Marlin/src/gcode/control/M80_M81.cpp

```diff
@@ -103,7 +103,10 @@ void GcodeSuite::M81() {
 
   #if HAS_SUICIDE
     stepper.synchronize();
-    suicide();
+    #ifndef suicide
+	#define suicide
+	#endif
+    suicide;
```

Comment:
Which platforms define the `suicide` macro? And what do they define it as? It's better in Marlin to call `kill()`, as this will also set the suicide pin.

Reply:
Gt2560 is the board that causes the error, I'll look through the pins after work, unless you're bored.

Reply:
I was confused, since you wrote `suicide;` rather than `suicide();`. You can't use the pre-processor directives `#ifdef` / `#ifndef` for function names, only for pre-processor macro names. I'll correct this PR and push the fixed version shortly.